docker/debian/plume/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM privyplace/debian:latest
+
+USER root
+
+ENV PLUME_VERSION=0.3.0-alpha-2
+
+RUN set -eux \
+    && apt-get update \
+    && apt-get install -y \
+                       libpq5 \
+                       libssl1.1 \
+                       libsqlite3-0 \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* \
+    && wget -q -O /tmp/www.tar.gz https://github.com/Plume-org/Plume/releases/download/${PLUME_VERSION}/plume-postgres.tar.gz \
+    && mkdir -p /www/search_index \
+    && chown -R www-data:www-data /www \
+    && tar xzfv /tmp/www.tar.gz -C /www \
+    && mv /www/bin/* /usr/local/bin \
+    && rm -f /tmp/www.tar.gz
+
+COPY run.sh /run.sh
+
+USER www-data
+
+WORKDIR /www
+
+EXPOSE 7878
+
+CMD ["/run.sh"]
```

Comment:
**Suggestion:** Consider using a multi-stage build to reduce the size of the final Docker image. This can be done by using a separate build stage to compile the application and then copying the compiled application into the final image.
```suggestion
# This is a simplified example and may not work without modification
FROM privyplace/debian:latest as build
USER root
ENV PLUME_VERSION=0.3.0-alpha-2
RUN set -eux \
    && apt-get update \
    && apt-get install -y \
                       libpq5 \
                       libssl1.1 \
                       libsqlite3-0 \
    && wget -q -O /tmp/www.tar.gz https://github.com/Plume-org/Plume/releases/download/${PLUME_VERSION}/plume-postgres.tar.gz \
    && tar xzfv /tmp/www.tar.gz -C /www \
    && mv /www/bin/* /usr/local/bin \
    && rm -f /tmp/www.tar.gz

FROM privyplace/debian:latest
COPY --from=build /www /www
COPY run.sh /run.sh
USER www-data
WORKDIR /www
EXPOSE 7878
CMD ["/run.sh"]
```